ICA.md

```diff
@@ -290,22 +290,21 @@ relationship;
 
 6.4.2. Contractor, under no circumstances is entitled to receive from Customer
 any benefits or insurance, including without limitation any unemployment,
-workers compensation or disability benefits, vacation or sick pay; 
+worker's compensation or disability benefits, vacation or sick pay; 
 
 6.4.3. Contractor is solely responsible for determining the method and means
 by which Customer will accomplish the Services and otherwise fulfill
 Contractor's obligations hereunder; and 
 
 6.4.4. Contractor will be solely responsible for the professional performance
-of the Services, and will receive no assistance, direction or control from
+of the Services, and will receive no assistance, direction, supervision or control from
```

Comment:
> and will receive no assistance, direction

does that mean I can't recommend insurance as we discussed?

Reply:
I think that this solely relates to the 'professional performance' i.e. how the job gets done. Making a recommendation  for the insurance (with the clarification that it's not being forced on them at all) should be fine.